versioned_docs/version-3.13/whats-new.md

```diff
@@ -0,0 +1,81 @@
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+import {
+  RabbitMQServerVersion,
+} from '@site/src/components/RabbitMQServer';
+
+import MQTTLogo from '@site/static/img/mqtt-logo/mqtt-ver.svg';
+import KhepriLogo from '@site/static/img/khepri-logo.svg';
+
+# What's New in RabbitMQ 3.13
+
+RabbitMQ 3.13 is a new release branch [published on 22 Feb
+2024](/blog/2024/03/11/rabbitmq-3.13.0-announcement). <strong>RabbitMQ
+<RabbitMQServerVersion/></strong> is the latest patch release for this release
+branch. See the [Release Information section](/release-information) to learn
+more about all patch releases for this release branch.
+
+## MQTT 5.0 support
+
+<MQTTLogo className="floating-logo"/>
+
+MQTT is the standard protocol for the Internet of Things (IoT).
+
+IoT remote devices can have a poor network quality when connecting to the
+broker. Therefore, MQTT is lightweight: MQTT protocol headers are small to save
+network bandwidth. Since IoT devices might often disconnect and reconnect -
+imagine a car driving through a tunnel - MQTT is also efficient: Clients are
+connected and authenticated with a handshake that is shorter compared to other
+messaging protocols.
+
+[MQTT 5.0](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html) is
+the latest version of the protocol and [is now supported by
+RabbitMQ 3.13.0](/blog/2023/07/21/mqtt5)!
+
+## Stream filtering
+
+[Stream filtering](/blog/2023/10/16/stream-filtering) allows to save bandwidth
+between the broker and consuming applications when those applications need only
+a subset of the messages of a stream.
+
+## Classic Queue Performance Improvements
+
+Classic queues can use [the newer message store
+v2](/blog/2024/01/11/3.13-release) which brings significant performance
+improvements, in terms of publishing/consuming rates, latency and memory
+footprint.
+
+## New Experimental Metadata Database
+
+<KhepriLogo className="floating-logo"/>
+
+[Khepri](https://www.youtube.com/watch?v=whVqpgvep90) is a new storage backend
+for RabbitMQ metadata that is designed to replace Mnesia. It is not yet *ready
+for production use* but we encourage users to try it out in test environments
+and provide feedback.
+
+Our plan is to completely remove Mnesia in the future. This should
+significantly improve RabbitMQ tolerance to network partitions. Once we switch
+to Khepri, there will be no partition handling strategy configuration
+(`pause_minority`, `autoheal`, and so on) — Khepri is based on the Raft
+protocol, just like quorum queues and therefore the semantics of what to do
+when a partition occurs are well defined and not configurable.
```

Comment:
This section is very good. How about we add Khepri into the title " New Experimental Metadata Database: Khepri"

do we need "and therefore" in the last line........ how about just "Khepri is based on the Raft
protocol, just like quorum queues, the semantics of what to do
when a partition occurs are well defined and not configurable."

Reply:
Are we documenting anything about "Message Containers" in the "What's new? I think this is more of an internal feature is it?

Reply:
How about: "Just like quorum queues, Khepri is also based on Raft protocol. As a result, the semantics of what to do when a partition occurs are well defined and not configurable."

Reply:
Thank you, applied!